src/variant_detection/variant_output.cpp

```diff
@@ -0,0 +1,81 @@
+#include <iostream> // for std::cout
+
+#include "structures/junction.hpp"              // for class Junction
+#include "variant_detection/variant_output.hpp"
+#include "variant_parser/variant_record.hpp"    // for class variant_header
+
+
+void find_and_output_variant(std::vector<Cluster> const clusters, std::ostream & out_stream)
+{
+    variant_header header{};
+    header.set_fileformat("VCFv4.3");
+    header.add_meta_info("SVTYPE", 1, "String", "Type of SV called.", "iGenVarCaller", "1.0");
+    header.add_meta_info("SVLEN", 1, "Integer", "Length of SV called.", "iGenVarCaller", "1.0");
+    header.add_meta_info("END", 1, "Integer", "End position of SV called.", "iGenVarCaller", "1.0");
+    header.print(out_stream);
+    for (size_t i = 0; i<clusters.size(); ++i)
```

Comment:
```suggestion
    for (size_t i = 0; i < clusters.size(); ++i)
```
Just a small spacing consistency.